handbook/engineering/distribution/goals.md

```diff
@@ -11,9 +11,15 @@ Creating a new release for our deployments is currently a semi-automated process
 - **Owner**: Distribution Team
 - **Status**: In Progress
 - **Outcomes**:
-  - Releases can be triggered by a single manual step
-  - All supported deployment types are released at the same time with the same command
-  - Support documentation enables any engineer to perform a release with confidence
+  - Releases can be triggered by a single manual step.
+  - All supported deployment types are released at the same time with the same command.
+  - Support documentation enables any engineer to perform a release with confidence.
+- **Milestones**:
+  - ~~Enable running e2e test on an environment matching its test (docker-compose in a VM).~~
+  - Enable continuos e2e tests on `main`.
```

Comment:
```suggestion
  - Enable continuous e2e tests on `main`.
```

Will they still be too slow to enable pre-merge? (if we can enable them pre-merge then we would not need to solve the notification problem for the next milestone)

Reply:
They take around 10m. I would not enable e2e tests on PR as those are generally long running tests, as an example, it would be desirable for me to `deploy gke cluster -> depoy to it -> test user flows` as well, and spinning all that infra takes time which would be really bad for PRs.